src/BenchmarkDotNet/Running/BenchmarkSwitcher.cs

```diff
@@ -50,73 +49,41 @@ public static BenchmarkSwitcher FromAssemblyAndTypes(Assembly assembly, Type[] t
         /// <summary>
         /// Run all available benchmarks.
         /// </summary>
-        public IEnumerable<Summary> RunAll() => Run(new[] { "*" });
+        public IEnumerable<Summary> RunAll() => Run(new[] { "--filter", ".*" });
```

Comment:
>.* [](start = 73, length = 2)

We are asking users for glob, but expect regex wildcard. We should be consistent here.

Reply:
@jorive I changed regex  into real glob